modules/ROOT/pages/_partials/proc_self-signed-certs.adoc

```diff
@@ -29,26 +28,25 @@ base64 --decode | sed -e '1,/^-----END RSA PRIVATE KEY-----$/ d'  > /tmp/localcl
 The resulting file is located in the `/tmp` directory.
 
 [[installing-on-device]]
-== Installing the OpenShift Root Certificate Authority Cert on a Device
+== 2. Installing the OpenShift Root Certificate Authority Cert on a Device
+
+[role="primary"]
+.Android
 
-. Configure a pin to ensure the certificate can be installed.
+****
+.. Enable PIN protection on the mobile device to ensure the certificate can be installed.
 
-. Copy the file to your device using one of the following methods:
+.. Copy the file to your device using one of the following methods:
 +
-* USB *(Android only)* - Attach the device to the machine hosting
+* USB - Attach the device to the machine hosting
 OpenShift via USB and drag the certificate to the devices file system
 * Email attachment - Simply email the certificate to an address
 accessible from the device and download the attachment.
 * Cloud service - Use a cloud storage service such as Dropbox or Google
 Drive which is accessible from the device, browse to the certificate and
 proceed to download it.
-
-. Configure your device:
-
-[role="primary"]
-.Android
-
-****
+.. Configure your device:
++
 Go to Settings > Security > Install from storage. It should detect the
```

Comment:
Again, is there a doc in Android docs that we can link to which explains this in more detail?